lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -272,6 +315,35 @@ private function hydrateGenericField(
         }
     }
 
+    /**
+     * Hydrate "json_array" field types.
+     *
+     * @param NodeInterface $node
+     * @param DocumentAccessor $accessor
+     * @param mixed $fieldName
+     * @param mixed $locale
+     * @param array $fieldMapping
+     */
+    private function hydrateJsonArrayField(
+        NodeInterface $node,
+        DocumentAccessor $accessor,
+        $fieldName,
+        $locale,
+        array $fieldMapping
+    ) {
+        try {
+            $phpcrName = $this->encoder->encode($fieldMapping['encoding'], $fieldMapping['property'], $locale);
+            $value = $node->getPropertyValueWithDefault(
+                $phpcrName,
+                $this->getDefaultValue($fieldMapping)
+            );
+            $accessor->set($fieldName, json_decode($value, true));
+        } catch (InvalidLocaleException $ex) {
+            // arguments unvalid no valid propertyname could be generated (e.g. no locale given for localized encoding)
```

Comment:
like i said this is "copied" from the other types we should add an issue and fix that in another PR